public/apps/configuration/panels/auth-view/authentication-sequence-panel.tsx

```diff
@@ -109,6 +110,7 @@ export function AuthenticationSequencePanel(props: { authc: [] }) {
         pagination={true}
         sorting={true}
         search={search}
+        message={props.loading ? loadingSpinner : items.length === 0 && noItemsFoundMsg}
```

Comment:
Would suggest have a function for it since it happens several times.

Reply:
Created function and refactored the code.